CONTRIBUTING.md

```diff
@@ -41,10 +42,9 @@ Further code convention adopted
 ```
 - No empty line after class declaration
 - No empty line after `<?php`
-- Last element of array must end with coma
-- One empty line after the dockblock header
-- Array and use statement must be written in alphabetical order
+- Last element of a multi-line array must be followed by a comma
+- One empty line after the docblock header
+- Array elements and `use` statements should be written in a logical order, preferrably alphabetically
```

Comment:
I think forcing an alphabetic order can be very inconvenient sometimes, like when certain elements are easier to read when grouped.
 
For example:
```
$methods = [
  'show_user', 'add_user', 'edit_user', 'delete_user',
  'show_group', 'add_group', 'edit_group', 'delete_group',
];
```
vs
```
$methods = [
  'add_group', 'add_user', 'delete_group', 'delete_user',
  'edit_group', 'edit_user', 'show_group', 'show_user',
];
```

The first example is easier to follow: top row `user`, bottom row `group`
The second one (alphabetical) is a bit more difficult.

(Might not be the best example, but I hope it illustrates the point I'm trying to make :smiley:)